javascript-data-model/order-history.js

```diff
@@ -0,0 +1,57 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    totalPrice: 34.00,
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-8772232',
+    deliveredDate: 'August 8, 2020',
+    handedToResident: false,
+    title: 'JavaScript for impatient programmers',
+    hasAuthor: true,
+    returnWindowDate: 'September 7, 2020',
+    originalPrice: 31.55,
+    writtenProductReview: false
+  },
```

Comment:
In this case, the data for the order itself is intermixed with the data concerning the items ordered, which makes this data model ambiguous and difficult to use. You should have a property of the object for the order that contains the data for the item(s) ordered by the user.